### DISCUSSION TITLE
Public Enseignant·e·s

### DISCUSSION THREAD

**Discussion by LHuberson** [unistra/esup-immersup-demandes#6] [3- Création de compte pour venir en immersion]
Bonjour, nous sommes amenés a organiser des actions à destination des enseignant·e·s. Nous aimerions pouvoir permettre les inscriptions de ce public. Il s'agit à l'heure actuelle de deux actions qui rentrent dans le PAF mais pour lequel les inscriptions aux ateliers sont gérés par le SUIO-IP. Les données nécessaires à l'inscription sont les Nom, Prénom, Établissement, Statut (Enseignant, Proviseur, Psy EN, PP), Mail.

**Comment by SandrineRobert** [comment]
Fonctionnalités qui nous intéresseraient également côté université de Bordeaux car nous aimerions proposer des immersions aux enseignants du secondaire (GT en cours)

**Comment by efinkbeiner5** [comment]
Solution envisagée : Utiliser le profil Visiteur en rajoutant des type de visiteurs paramétrables. Ajouter l'information de l'établissement d'origine (lycée ou université). L'avantage c'est que cela ne change pas les inscriptions en immersion, ni la gestion des pièces.
On peut imaginer une pièce commune devant justifier le profil. 
Chiffrage : fort

**Comment by efinkbeiner5** [comment]
le sujet reste ouvert, non priorisé en 2024.

**Comment by MagaliMoserNoriaIRL** [comment]
Nous sommes également intéressés par cette fonctionnalité en Alsace :)

**Comment by efinkbeiner5** [comment]
La justification ou non du profil pourrait être paramétrable dans le type de visiteur. (un peu comme les types d'annulation). 
On pourrait proposer des restrictions des créneaux sur le type de visiteur. 
Se recoupe avec #50

**Comment by efinkbeiner5** [comment]
Utiliser le profil visiteur en rajoutant des types de visiteurs paramétrables (Enseignants, PsyEn,Proviseurs, élève d'un lycée non conventionné, reprise d'étude…) --> Etude aussi de la possibilité d'utiliser la fédération d'identité Agent du secondaire

1 - ajout d'une table dans l'admin avec le libellé du type de visiteur
2 - dans la table des pièces justificatives, quand on choisit que cela concerne un visiteur, on peut ajouter les types de visiteurs concernés.
3 - ajout d'un type sur le visiteur à la création du compte
4 - changement de la fiche visiteur pour demander les bonnes pièces
Pas de fédération d'identité même dans le cas des agents du secondaire

Complexité : Forte
Chiffrage : 6 jours

**Comment by efinkbeiner5** [comment]
Réalisé en 2025 avec les types de visiteurs
